From 7c08e556b0c6503176b57082ca120518cf011d9d Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Tue, 24 Dec 2024 12:34:38 -0800
Subject: [PATCH 2/2] Remove dependency on #include <numbers>

---
 src/common/bspf.hxx | 9 ++++-----
 1 file changed, 4 insertions(+), 5 deletions(-)

diff --git a/src/common/bspf.hxx b/src/common/bspf.hxx
index 180fd97f3..6f16315ae 100644
--- a/src/common/bspf.hxx
+++ b/src/common/bspf.hxx
@@ -55,7 +55,6 @@ using uInt64 = uint64_t;
 #include <cctype>
 #include <cstdio>
 #include <ctime>
-#include <numbers>
 #include <ranges>
 #include <utility>
 #include <vector>
@@ -120,10 +119,10 @@ static const string EmptyString;
 
 namespace BSPF
 {
-  static constexpr float PI_f = std::numbers::pi_v<float>;
-  static constexpr double PI_d = std::numbers::pi_v<double>;
-  static constexpr double ln10 = std::numbers::ln10;
-  static constexpr double ln2 = std::numbers::ln2;
+  static constexpr float PI_f = 3.14159265358979323846f;
+  static constexpr double PI_d = 3.14159265358979323846;
+  static constexpr double ln10 = 2.30258509299404590109;
+  static constexpr double ln2  = 0.693147180559945309417;
 
   // CPU architecture type
   // This isn't complete yet, but takes care of all the major platforms
-- 
2.47.1

